tests/pstl-tests/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+function(compile_test source)
+    set(test_name "pstl.${source}")
+    set(source "${source}.cpp")
+    add_executable(${test_name} ${source})
+    target_link_libraries(${test_name} PUBLIC ${gtest_BINARY_DIR}/libgtest.a
+                                       PUBLIC ${gtest_BINARY_DIR}/libgtest_main.a
+                                       PUBLIC pthread)

```

Comment:
`Threads::Threads` is more portable than `pthread`

Reply:
Fixed.